apps/web/app/api/auth/oauth/refreshToken/route.ts

```diff
@@ -50,19 +50,31 @@ async function handler(req: NextRequest) {
     return NextResponse.json({ message: "Unauthorized" }, { status: 401 });
   }
 
-  const payload: OAuthTokenPayload = {
+  const payloadAuthToken: OAuthTokenPayload = {
     userId: decodedRefreshToken.userId,
     teamId: decodedRefreshToken.teamId,
     scope: decodedRefreshToken.scope,
     token_type: "Access Token",
     clientId: client_id,
   };
 
-  const access_token = jwt.sign(payload, secretKey, {
+  const payloadRefreshToken: OAuthTokenPayload = {
+    userId: decodedRefreshToken.userId,
+    teamId: decodedRefreshToken.teamId,
+    scope: decodedRefreshToken.scope,
+    token_type: "Refresh Token",
+    clientId: client_id,
+  };
+
+  const access_token = jwt.sign(payloadAuthToken, secretKey, {
     expiresIn: 1800, // 30 min
   });
 
-  return NextResponse.json({ access_token }, { status: 200 });
+  const refresh_token = jwt.sign(payloadRefreshToken, secretKey, {
+    expiresIn: 30 * 24 * 60 * 60, // 30 days
+  });
+
+  return NextResponse.json({ access_token, refresh_token }, { status: 200 });
```

Comment:
This code is new to me so I have a question - if we are passing oAuth client's secret to this handler and then returning `refresh_token`, then we should update `oAuthClient.clientSecret` by setting it to the new `refresh_token` on line 73, right? aka current "clientSecret" is passed to handler, we create new one and we have to store it in database so that next time this call happens with the new "refresh_token" the clientSecret call does not fail
```
const client = await prisma.oAuthClient.findFirst({
    where: {
      clientId: client_id,
      clientSecret: hashedSecret,
    },
    select: {
      redirectUri: true,
    },
  });
```
?